packages/ia-components/sandbox/theatres/components/audio-player/audio-player-main.jsx

```diff
@@ -39,11 +40,14 @@ export default class TheatreAudioPlayer extends Component {
 
     this.state = {
       urlSetterFN: null,
+      mediaSource: 'player'
     };
 
     this.showMedia = this.showMedia.bind(this);
     this.createTabs = this.createTabs.bind(this);
     this.receiveURLSetter = this.receiveURLSetter.bind(this);
+    this.toggleMediaSource = this.toggleMediaSource.bind(this);
+    this.showLinerNotes = this.showLinerNotes.bind(this);
```

Comment:
**Suggestion**: With this many bind calls, can we create an iterator to bind these and any future ones? Something like:

```
["showMedia", "createTabs", ...].forEach((method) => this[method].bind(this));
```